grc/gui/canvas/block.py

```diff
@@ -158,7 +158,7 @@ def create_labels(self, cr=None):
         # update the params layout
         if not self.is_dummy_block:
             markups = [param.format_block_surface_markup()
-                for param in self.params.values() if (param.hide not in ('all', 'part') or (param.dtype == 'id' and force_show_id))]
+                for param in self.params.values() if (param.hide not in ('all', 'part'))]
```

Comment:
I like this change, as it remove logic in the presentation layer which could have easily been handled via `param.hide` in the first place. Note, that you'll probably also what to remove `force_show_id = ...` a few lines above, as it is no longer needed.

Nitpick: if this line is touched it deserves to get proper indentation...

Reply:
Removed and I did what I think is proper indentation (basically every python beautifier disagreed with this and each other though...if youd like a change can you be more specific?).